docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
 ---
-title: How to Contribute to Teleport's Documentation
-description: Follow this guide to get started contributing changes to Teleport's documentation
+title: How to contribute to Teleport documentation
+description: Follow this guide to learn how to contribute changes to Teleport documentation.
 ---
 
-This guide describes the general workflow for making documentation changes.
-It assumes you already have Node >=14 and `yarn` installed.
+We encourage everyone with an interest in improving Teleport documentation to contribute content, submit issues, and suggest changes.
+This guide describes the process for making changes to the documentation whether you are an internal or external contributor.
 
-## Step 1/4 Set up your local environment
+## Before you begin
```

Comment:
Should we title this section "Prerequisites" to be consistent with other how-to guides in the docs?

Reply:
Yeah, I see that most of the doc uses the more formal wording.